azure/azure-release-template.yml

```diff
@@ -65,62 +69,74 @@ extends:
-          url="https://$APIGEE_ENVIRONMENT.api.service.nhs.uk/$SERVICE_BASE_PATH/_ping"
+          url="https://$APIGEE_ENVIRONMENT.api.service.nhs.uk/$SERVICE_BASE_PATH/_status"
           interval_in_seconds=5
           releaseCommit="$(Build.SourceVersion)"
-          path=".commitId"
-          printf "\nPolling '$url' every $interval_in_seconds seconds, until commit is: '$releaseCommit'\n"
-          while true;
+          printf "\nPolling %s every %s seconds, until commit is: %s\n" "$url" "$interval_in_seconds" "$releaseCommit"
+          attempts=0
+          success=0
+          until [ $attempts -eq 15 ]
           do
-            deployedCommit=`curl $url -s | jq -r $path`
-            if [[ "$deployedCommit" == "$releaseCommit" ]]; then
+          responseData=$(curl -H "apiKey: $(status-endpoint-api-key)" "$url" -s)
+          deployedCommit=$(echo $responseData | jq -r ".commitId")
+          validatorUp=$(echo $responseData | jq -r ".validator")
+          if [ "$deployedCommit" == "$releaseCommit" ] && [ "$validatorUp" == true ]; then
+              success=1
               break;
-            fi;
-            sleep $interval_in_seconds;
+          fi;
+          ((attempts=attempts+1))
+          sleep $interval_in_seconds;
           done
 
+          if [ $success == 0 ]
+          then
+              echo "Smoke tests failed, API was not ready in time"
+              exit 255
+          fi
+
           # Verify
           chmod +x ./broker/verify.ts
           make verify-pacts
 
           # Tag
           docker pull pactfoundation/pact-cli:latest > /dev/null
-          docker run --rm -e PACT_BROKER_BASE_URL=$PACT_BROKER_URL -e PACT_BROKER_USERNAME=$PACT_BROKER_BASIC_AUTH_USERNAME -e PACT_BROKER_PASSWORD=$PACT_BROKER_BASIC_AUTH_PASSWORD pactfoundation/pact-cli:latest broker create-version-tag --pacticipant=$PACT_PROVIDER+convert+$PACT_VERSION --version=$PACT_VERSION --tag=$APIGEE_ENVIRONMENT
-          docker run --rm -e PACT_BROKER_BASE_URL=$PACT_BROKER_URL -e PACT_BROKER_USERNAME=$PACT_BROKER_BASIC_AUTH_USERNAME -e PACT_BROKER_PASSWORD=$PACT_BROKER_BASIC_AUTH_PASSWORD pactfoundation/pact-cli:latest broker create-version-tag --pacticipant=$PACT_PROVIDER+prepare+$PACT_VERSION --version=$PACT_VERSION --tag=$APIGEE_ENVIRONMENT
-          docker run --rm -e PACT_BROKER_BASE_URL=$PACT_BROKER_URL -e PACT_BROKER_USERNAME=$PACT_BROKER_BASIC_AUTH_USERNAME -e PACT_BROKER_PASSWORD=$PACT_BROKER_BASIC_AUTH_PASSWORD pactfoundation/pact-cli:latest broker create-version-tag --pacticipant=$PACT_PROVIDER+process+$PACT_VERSION --version=$PACT_VERSION --tag=$APIGEE_ENVIRONMENT
+          docker run --rm -e PACT_BROKER_BASE_URL="$PACT_BROKER_URL" -e PACT_BROKER_USERNAME="$PACT_BROKER_BASIC_AUTH_USERNAME" -e PACT_BROKER_PASSWORD="$PACT_BROKER_BASIC_AUTH_PASSWORD" pactfoundation/pact-cli:latest broker create-version-tag --pacticipant=$PACT_PROVIDER+convert+"$PACT_VERSION" --version="$PACT_VERSION" --tag="$APIGEE_ENVIRONMENT"
```

Comment:
I'd put the double quotes around the entirety of `$PACT_PROVIDER+convert+$PACT_VERSION`. Similarly on the other `docker run` lines